src/components/transactions/explorerTransactions/explorerTransactions.js

```diff
@@ -125,7 +125,7 @@ class ExplorerTransactions extends React.Component {
       activeToken: this.props.activeToken,
       transactions: this.props.transactions.data.data,
       activeFilter: this.props.transactions.urlSearchParams.filters.direction,
-      loading: this.props.transactions.isLoading ? [actionTypes.transactionsLoaded] : [],
+      loading: this.props.transactions.isLoading ? [actionTypes.getTransactionsSuccess] : [],
```

Comment:
Should be the same action as in https://github.com/LiskHQ/lisk-hub/blob/da0946a4f0b54281e404d732cc2fb501c6bf5904/src/actions/transactions.js#L65
```suggestion
      loading: this.props.transactions.isLoading ? [actionTypes.getTransactions] : [],
```